docs/source/miniconda.rst

```diff
@@ -1,42 +1,58 @@
- |3.7   | installer) <https: | installer) <https: | installer) <https: |
- |      | //repo.anaconda.co | //repo.anaconda.co | //repo.anaconda.co |
- |      | m/miniconda/Minico | m/miniconda/Minico | m/miniconda/Minico |
- |      | nda3-latest-Window | nda3-latest-MacOSX | nda3-latest-Linux- |
- |      | s-x86_64.exe>`__   | -x86_64.sh>`__     | x86_64.sh>`__      |
- |      |                    |                    |                    |
- |      | `32-bit (exe       | `64-bit (.pkg      | `32-bit (bash      |
- |      | installer) <https: | installer) <https: | installer) <https: |
- |      | //repo.anaconda.co | //repo.anaconda.co | //repo.anaconda.co |
- |      | m/miniconda/Minico | m/miniconda/Minico | m/miniconda/Minico |
- |      | nda3-latest-Window | nda3-latest-MacOSX | nda3-latest-Linux- |
- |      | s-x86.exe>`__      | -x86_64.pkg>`__    | x86.sh>`__         |
- +------+--------------------+--------------------+--------------------+
- |Python| `64-bit (exe       | `64-bit (bash      | `64-bit (bash      |
- |2.7   | installer) <https: | installer) <https: | installer) <https: |
- |      | //repo.anaconda.co | //repo.anaconda.co | //repo.anaconda.co |
- |      | m/miniconda/Minico | m/miniconda/Minico | m/miniconda/Minico |
- |      | nda2-latest-Window | nda2-latest-MacOSX | nda2-latest-Linux- |
- |      | s-x86_64.exe>`__   | -x86_64.sh>`__     | x86_64.sh>`__      |
- |      |                    |                    |                    |
- |      | `32-bit (exe       | `64-bit (.pkg      | `32-bit (bash      |
- |      | installer) <https: | installer) <https: | installer) <https: |
- |      | //repo.anaconda.co | //repo.anaconda.co | //repo.anaconda.co |
- |      | m/miniconda/Minico | m/miniconda/Minico | m/miniconda/Minico |
- |      | nda2-latest-Window | nda2-latest-MacOSX | nda2-latest-Linux- |
- |      | s-x86.exe>`__      | -x86_64.pkg>`__    | x86.sh>`__         |
- +------+--------------------+--------------------+--------------------+
-
- `Installation
- instructions <https://conda.io/projects/conda/en/latest/user-guide/install/index.html>`__
-
- Other resources:
+Miniconda is a free minimal installer for conda. It is a small, bootstrap
+version of Anaconda that includes only conda, Python, the packages they depend
+on, and a small number of other useful packages, including pip, zlib and a
+few others. Use the ``conda install command`` to install 720+ additional conda
+packages from the Anaconda repository.
+
+`See if Miniconda is right for you <https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html#anaconda-or-miniconda>`_.
+
+Windows installers
+==================
+
+.. csv-table:: Windows
+   :header: Python version,Name,Size,SHA256 hash
+   :widths: 5, 10, 5, 80
+
+   Python 3.7,`Miniconda3 Windows 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe>`_,76.2 MiB,``a3a8921c2dec37f4ef37b9fa7b337dba237ccacec56bed3d8b8c300ed852c84f``
```

Comment:
Unfortunately, the sizes are off.  I'll fix in a commit shortly if I have access.

1 MiB = 1024 * 1024 bytes; 1 MB = 1000 * 1000 bytes; OS X shows you MB in the finder view.

Should be either 72.6MiB or 76.2MB.  (76157168 bytes)